dj_hetmech/settings.py

```diff
@@ -44,12 +44,22 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'django_extensions',
+    'rest_framework',
+    'corsheaders',
     'dj_hetmech_app',
 ]
 
+REST_FRAMEWORK = {
+    'PAGE_SIZE': 25,
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',
+    'DEFAULT_RENDERER_CLASSES': ('rest_framework.renderers.JSONRenderer', ),
+    'DEFAULT_FILTER_BACKENDS': ('django_filters.rest_framework.DjangoFilterBackend',)
```

Comment:
missing comma at end of line

Reply:
@dhimmel Both `search` and `limit` parameters are standard in `django-rest-framework`, if you need other filters, let me know and I will add them.

Reply:
I am mostly interested in how you know which parameters are available, where that gets configured, and seeing the corresponding `django-rest-framework` documentation.

Reply:
I added some comments with relevant DRF documentation URL.